docs/ADRs/0002-code-platform.md

```diff
@@ -0,0 +1,62 @@
+---
+    title: 0002 Code Platform
+    adr:
+        author: Berry den Hartog
+        created: 28-Feb-2024
+        status:  accepted
+---
+
+## Context
+
+In the landscape of software development, the choice of coding platform significantly impacts developer productivity,
+collaboration, and code quality. it's crucial to evaluate and select a coding platform that aligns with our development
+needs and fosters efficient workflows.
+
+## Assumptions
+
+The following assumptions are made:
+
+- our work should be visible to external teams to promote transparency and facilitate collaboration.
+- The coding platform should be easily available for developers
+- the coding platform should offert collaboration tools between developers and the community
+- the coding platform should offert security and dependecy management tools
+- the coding platform should have low costs
```

Comment:
A good Cost/Efficiency trade-off, as we are currently also debating where we would opt for the paid version of Github? Or would that be a new ADR?

Reply:
allthough I thinks this is a good point, it is open for debate 'what is low' and what is good 'cost/efficiency'.. so maybe this should be more politically rephrased (allthough it becomes a bit more meaningless as well):
- the pricing model should be suitable for our budget and needs

Reply:
i checked the licences. currenty it does give low benefits for a github licences